pkgs/applications/networking/instant-messengers/telegram/tdesktop/tg_owt.nix

```diff
@@ -27,6 +28,7 @@ stdenv.mkDerivation {
     openh264 usrsctp libevent libvpx
     libX11 libXtst libXcomposite libXdamage libXext libXrender libXrandr libXi
     glib abseil-cpp pcre util-linuxMinimal libselinux libsepol pipewire
+    mesa valgrind libepoxy libglvnd
```

Comment:
Where's these from? `valgrind` seems wrong here since it's a debugging or testing tool for memory leaking.

Seems `tg_owt` still builds without `valgrind`.

Reply:
It seems like `valgrind` was added here by mistake, because I can't remember adding it. `mesa` and `libepoxy` are from [wiki](https://github.com/telegramdesktop/tdesktop/wiki/The-Packaged-Building-Mode), `libglvnd` fixes the following `tg_owt` compilation error 
```
/nix/store/ky80nbab7nvdmq3glxmj4x30mqcfjnap-libepoxy-1.5.9-dev/include/epoxy/egl_generated.h:11:10: fatal error: EGL/eglplatform.h: No such file or directory
   11 | #include "EGL/eglplatform.h"
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```